xarray/backends/common.py

```diff
@@ -535,6 +536,22 @@ def open_datatree(
 
         raise NotImplementedError()
 
+    def open_groups_as_dict(
+        self,
+        filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+        **kwargs: Any,
```

Comment:
Maybe we should not make the same mistake as with `open_dataset` and prevent liskov errors.

```suggestion
```

If the abstract method supports any kwargs, so must all subclass implementations, which is not what we want.

Reply:
@headtr1ck from my understanding I think the `**kwargs` were added back to fix this issue #9135

Reply:
Hmmm, Not sure.
But since this is the same problem in all other backend methods, I'm fine with leaving it as it is (and possibly change it in a future PR all together).

Reply:
Sounds good we can revisit this on another PR.